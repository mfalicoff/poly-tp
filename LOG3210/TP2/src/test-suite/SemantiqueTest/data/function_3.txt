Declaration

EndDeclaration

num a(num b, num c){
    c = z;
    return c;
}