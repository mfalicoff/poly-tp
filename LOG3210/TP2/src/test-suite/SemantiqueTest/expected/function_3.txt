Invalid type in assignation of Identifier c